src/test/java/io/confluent/connect/elasticsearch/integration/ElasticsearchSinkTaskIT.java

```diff
@@ -27,12 +27,32 @@
 import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.Struct;
 import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
+import org.slf4j.MDC;
 
 @Category(IntegrationTest.class)
 public class ElasticsearchSinkTaskIT extends ElasticsearchIntegrationTestBase {
 
+  private ElasticsearchSinkTask task = new ElasticsearchSinkTask();
+
+  @Before
+  public void beforeEach() {
+    //MDC.put("connector.context", "[MyConnector|task1] ");
```

Comment:
Should this be active for the test? The remove is uncommented, and I'm assuming that's a no-op if the parameter is not configured.

Reply:
Will fix.